### PR TITLE
add cross circle icon

### PR DESCRIPTION
**Pull request by ethaMont**

### CODE REVIEW

**Comment by ethaMont** [comment]
ping

**Comment by tildedave** [comment]
lgtm

File: app/components/CrossCircle.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  render() {
+    return (
+      <svg {...this.props} className={this.getClassNames() } viewBox="0 0 144 144">
+        <title>CrossCircle</title>
+        <g className="tilt-icon-main">
+          <g transform="translate(-303.000000, -191.000000)" fill="#FFFFFF">
+            <g transform="translate(303.000000, 191.000000)">
+              <g className="cross" transform="translate(43.000000, 42.000000)">
+                <polygon id="Shape" points="0.779619771 3.6080469 56.5530642 59.3814913 59.3814913 56.5530642 3.6080469 0.779619771"></polygon>
+                <polygon id="Shape" points="56.390342 0.781230882 0.581453104 56.5901198 3.40988023 59.4185469 59.2187691 3.60965801"></polygon>
```
Comment (BrettBukowski) [inline app/components/CrossCircle.js]: remove the id attributes?

Comment (ethaMont) [inline app/components/CrossCircle.js]: oops
https://github.com/Crowdtilt/tilt-images/pull/67

**Pull request merged**